README.md

```diff
@@ -54,16 +54,20 @@ for more details.
 
 ## Release Notes
 
+### 0.5.3 - 2023-03-17
+
+- Added model formatting via Smithy Language Server 0.2.3. [#67](https://github.com/awslabs/smithy-vscode/pull/67)
+
 ### 0.5.2 - 2022-12-27
 
-- Updated to use [Smithy Language Server 0.2.2](https://github.com/awslabs/smithy-language-server/)
+- Updated to use [Smithy Language Server 0.2.2](https://github.com/awslabs/smithy-language-server/).
 
 ### 0.5.1 - 2022-09-29
 
-- Updated to use [Smithy Language Server 0.2.1](https://github.com/awslabs/smithy-language-server/)
-- Stop switching focus to VSCode's output window when the extension receives a message [#55](https://github.com/awslabs/smithy-vscode/pull/55)
-- Allow setting the root path to be used by the extension [#54](https://github.com/awslabs/smithy-vscode/pull/54)
-- Update tests to use version 1.25.0 for Smithy packages [#56](https://github.com/awslabs/smithy-vscode/pull/56)
+- Updated to use [Smithy Language Server 0.2.1](https://github.com/awslabs/smithy-language-server/).
+- Stoped switching focus to VSCode's output window when the extension receives a message. [#55](https://github.com/awslabs/smithy-vscode/pull/55)
```

Comment:
Stopped?

Reply:
Fixed in [d5c1e8f](https://github.com/awslabs/smithy-vscode/pull/71/commits/d5c1e8fe0d7164d690d235510d89e295c63dbde5).